sycl/test-e2e/GroupAlgorithm/root_group.cpp

```diff
@@ -76,44 +78,45 @@ void testRootGroupFunctions() {
       sycl::ext::oneapi::experimental::use_root_sync};
 
   constexpr int testCount = 10;
-  bool *testResults = sycl::malloc_shared<bool>(testCount, q);
+  sycl::buffer<bool> testResults_buf{sycl::range{testCount}};
```

Comment:
I'd suggest using a consistent naming scheme. Since the code around this change uses camelCase, you could as well use that.